baseledger/x/proof/client/rest/txutil.go

```diff
@@ -85,15 +85,10 @@ func getKey(keyringInstance keyring.Keyring) (keyring.Info, error) {
 func NewKeyringInstance() (keyring.Keyring, error) {
 	input := &bytes.Buffer{}
 	fmt.Fprintln(input, viper.GetString("KEYRING_PASSWORD"))
+	fmt.Fprintln(input, viper.GetString("KEYRING_PASSWORD"))
```

Comment:
adding it 3 times because of prompt repeat (2 should be enough, but 3 just in case)

Reply:
also reminder, we need to use KEYRING_DIR and KEYRING_PASSWORD env variables

Reply:
viper is picking them up if we just go export VAR=value.. so should be fine